mullvad-version/build.rs

```diff
@@ -67,17 +69,18 @@ fn get_dev_suffix(target: Target, product_version: &str) -> String {
 
     // Get the git commit hashes for the latest release and current HEAD
     let product_version_commit_hash = git_rev_parse_commit_hash(&release_tag);
-    let current_head_commit_hash =
-        git_rev_parse_commit_hash("HEAD").expect("HEAD must have a commit hash");
+    let current_head_commit_hash = git_rev_parse_commit_hash("HEAD");
```

Comment:
You can just add a `?` here to short circuit and early-return `None` if we fail to get the hash for `HEAD`.